transport/http/handler.go

```diff
@@ -44,15 +44,17 @@ func popHeader(h http.Header, n string) string {
 
 // handler adapts a transport.Handler into a handler for net/http.
 type handler struct {
-	router      transport.Router
-	tracer      opentracing.Tracer
-	grabHeaders map[string]struct{}
+	router            transport.Router
+	tracer            opentracing.Tracer
+	grabHeaders       map[string]struct{}
+	bothResponseError bool
 }
 
 func (h handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	responseWriter := newResponseWriter(w)
 	service := popHeader(req.Header, ServiceHeader)
 	procedure := popHeader(req.Header, ProcedureHeader)
+	bothResponseError := fromAcceptValue(popHeader(req.Header, AcceptsBothResponseErrorHeader))
```

Comment:
`acceptsBothResponseError` might be more clear.

Reply:
The thing here is that this matches the field I have in `handler, inbound, outbound`, which I want to keep that way. Below will be `acceptsBothResponseError && h.bothResponseError` which is weird to me.